stock_batch_picking/i18n/stock_batch_picking.pot

```diff
@@ -0,0 +1,395 @@
+# Translation of Odoo Server.
```

Comment:
My opinion on this have changed since that issue, because I found somebody translating from an outdated POT file.